pkg/crypto/webkms/remotecrypto.go

```diff
@@ -103,6 +129,12 @@ const (
 	verifyMACURI  = "/verifymac"
 	wrapURI       = "/wrap"
 	unwrapURI     = "/unwrap"
+
+	// BBS+ endpoints.
```

Comment:
These should be generic rather than BBS specific.

Reply:
updated